Update AppWithPlugin tutorial to link to example. (#11017)
dotnet/docs: docs/core/tutorials/creating-app-with-plugin-support.md

## Other plugin examples in the sample

> There is a completed sample of this tutorial can be found in [the dotnet/samples repository](https://github.com/dotnet/samples/tree/master/core/extensions/AppWithPlugin). The completed sample includes a few other examples of `AssemblyDependencyResolver` behavior. For example, the `AssemblyDependencyResolver` object can also resolve native libraries included in NuGet packages as well as localized satellite assemblies. The `UVPlugin` and `FrenchPlugin` in the samples repository demonstrate these scenarios respectively.

Reviewer: ```suggestion
The complete source code for this tutorial can be found in [the dotnet/samples repository](https://github.com/dotnet/samples/tree/master/core/extensions/AppWithPlugin). The completed sample includes a few other examples of `AssemblyDependencyResolver` behavior. For example, the `AssemblyDependencyResolver` object can also resolve native libraries as well as localized satellite assemblies included in NuGet packages. The `UVPlugin` and `FrenchPlugin` in the samples repository demonstrate these scenarios.
```
